taxonomies/labels.txt

```diff
@@ -798,7 +798,7 @@ bg:Без пшеница
 de:Ohne Weizen
 nl:Zonder graan
 
-en:No gluten, Gluten-free, Without gluten, Gluten-free-food, free of gluten, Naturally gluten free, Gluten free, ni gluten
```

Comment:
For allergens, antibiotics etc. the ingredients parsing code expects the "No something" to be the canonical entry.

There's a comment about that at the top of the file:

# For labels that indicate the absence of something, in English, always make "No something" the first entry, and make "Something-free", "Without something" synonyms

Reply:
Maybe we can add that it's required by the ingredients parsing code.